packages/hoppscotch-sh-admin/src/pages/users/index.vue

```diff
@@ -572,45 +573,32 @@ const deleteUsers = async (id: string | null) => {
   const result = await usersDeletion.executeMutation(variables);
 
   if (result.error) {
-    const errorMessage =
-      result.error.message === DELETE_USER_FAILED_ONLY_ONE_ADMIN
-        ? t('state.delete_user_failed_only_one_admin')
```

Comment:
Admins can't be deleted and it's accounted for by the `ADMIN_CANNOT_BE_DELETED` error message check below. The `DELETE_USER_FAILED_ONLY_ADMIN` = `admin/only_one_admin_account_found` scenario applies to the case while removing the Admin status when there is only one Admin in the instance.

https://github.com/hoppscotch/hoppscotch/blob/0a71783eaaba4eac47977fc1d4da9a9a8f6080dd/packages/hoppscotch-backend/src/errors.ts#L6-L11

https://github.com/hoppscotch/hoppscotch/blob/0a71783eaaba4eac47977fc1d4da9a9a8f6080dd/packages/hoppscotch-backend/src/admin/admin.service.ts#L530-L543